### ISSUE TITLE
Add new terminology for "molecule"

### ISSUE THREAD

**Issue by Majpuc** [HumanBrainProject/openMINDS_instances#15]
I lack the possibility to be able to have different molecules as study target. Similarly to "cell type" I suggest that we have the instance "molecule" where the entries could be a set of controlled terms as follows:

Glutamate (Glu)
Acetylcholine (ACh)
Histamine
Dopamine (DA)
Norepinephrine (NE); also known as noradrenaline (NAd)
Epinephrine (Epi); also known as adrenaline (Ad)
gamma-Aminobutyric acid (GABA)
Serotonin (5-HT)
Dopamine (DA)
Oxytocin (Oxt)
Vasopressin; also known as antidiuretic hormone (ADH) 
 
They have of course properties, like excitatory, inhibitory, modulatory but this info could be added later.

**Comment by lzehl** [comment]
@Majpuc are these all proteins?

**Comment by Majpuc** [comment]
No, not only proteins. I mentioned neurotransmitters here in the first round, but actually also need controlled term lists for receptors, transporters, ions, etc.. 
I think we could have an umbrella term like "molecule" allowing to include all these "study target" without going into too much details. Open to suggestions and discussion

**Comment by lzehl** [comment]
molecules is a bit too general I would say. I would not mind maybe splitting to:
+ neurotransmitters
+ receptors
+ ionChannels
+ ions
+ hormons

[I would do these separations because the controlledTerms and corresponding instances will be used across openMINDS metadata models and the distinction could be very nice for the in-depth models.]

Would that be a good start?

**Comment by Majpuc** [comment]
Yes, good idea. Let's define these lists or fetch them from somewhere :)

**Comment by lzehl** [comment]
I think we have several of those already in the old KG. I will see how many I can automatically generated until tomorrow. @Majpuc thanks again for making the request!

**Comment by lzehl** [comment]
Example for neurotransmitters:
+ dopamine (http://purl.obolibrary.org/obo/CHEBI_18243)

**Comment by amkvello** [comment]
Maybe use "compounds" instead of "molecules" as a general term? If we need a general term to group neurotransmitters, receptors and so on, that is :)

**Comment by amkvello** [comment]
..but this would not cover receptors, though

**Comment by Majpuc** [comment]
The plan is to make several lists of controlled terms as Lyuba suggested above. The one I proposed covers main neurotransmitters. Similar lists could be created for the other categories (receptors, transporters, ion channels, ions, hormones, etc..)

**Comment by lzehl** [comment]
@Majpuc I agree. @amkvello thanks for preparing this and as discussed: define some groups now and I can rearrange later if necessary. @Majpuc I might get back to you about the groupings then.

**Comment by Majpuc** [comment]
@lzehl was this topic moved to another github project?
